src/Analyzers/CSharp/Analyzers/NewLines/ConditionalExpressionPlacement/ConditionalExpressionPlacementDiagnosticAnalyzer.cs

```diff
@@ -37,7 +37,10 @@ private void AnalyzeTree(SyntaxTreeAnalysisContext context)
             if (option.Value)
                 return;
 
-            Recurse(context, option.Notification.Severity, context.Tree.GetRoot(context.CancellationToken));
+            var root = context.Tree.GetRoot(context.CancellationToken);
+            if (context.FilterSpan.HasValue)
+                root = root.FindNode(context.FilterSpan.GetValueOrDefault());
```

Comment:
nit, at this point consider an extension on SyntaxTree that gets you the node for a nullable filter.  If null, you get root, otherwise FindNode.